src/components/FullscreenContainer.tsx

```diff
@@ -13,7 +13,7 @@ const FullscreenContainer = ({ children }: Props): JSX.Element => (
     margin="auto"
     alignItems="center"
     justifyContent="space-between"
-    bgcolor="#f6f7fb"
+    bgcolor="rgba(0,0,0,0.08)"
```

Comment:
Do we want this color factored out somewhere as a constant?

Reply:
I don't think we should update this. Here is the only place we use this color, because it needs to be a transparent background to show the background pattern. All the other backgrounds are part of the palette.